client/container.go

```diff
@@ -106,3 +106,16 @@ func (client *APIClient) ContainerStartExec(execid string, config *types.ExecSta
 
 	return client.hijack("/exec/"+execid+"/start", url.Values{}, config, header)
 }
+
+func (client *APIClient) ContainerInfo(name string) (*types.Container, error) {
```

Comment:
Need to add comments to this function, since I do not think it will pass `golint` check, otherwise.

Reply:
Thanks @allencloud . Comments is necessary.
And I got a suggestion from @skoo87 , should add a new method `Get` to replace method `ContainerInfo`.